### PR TITLE
sending ether

### PR DESCRIPTION
**Pull request by GreatSoshiant**
In this PR we provided a new page on Stylus by example and described how to send ether on the Rust Stylus contracts

Closes STY-47

### CODE REVIEW

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[package]
+name = "stylus-hello-world"
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:2]:
```suggestion
name = "sending_ether"
```

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/OffchainLabs/stylus-hello-world"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+description = "Stylus hello world example"
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:9]:
```suggestion
description = "Stylus sending ether example"
```

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+[package]
+name = "stylus-hello-world"
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/OffchainLabs/stylus-hello-world"
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:6]: We can change the homepage here to stylus-by-example or simply delete it

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+[dev-dependencies]
+tokio = { version = "1.12.0", features = ["full"] }
+ethers = "2.0"
+eyre = "0.6.8"
+
+[features]
+export-abi = ["stylus-sdk/export-abi"]
+debug = ["stylus-sdk/debug"]
+
+[[bin]]
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:27]: We probably don't need this

File: src/app/basic_examples/sending_ether/page.mdx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        Ok(())
+    }
+}
+```
+
+### `Cargo.toml`
+
+```toml
+[package]
+name = "stylus-hello-world"
```
Comment (Jason-W123) [inline src/app/basic_examples/sending_ether/page.mdx:122]:
```suggestion
name = "stylus-sending-ether"
```

File: src/app/basic_examples/sending_ether/page.mdx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/OffchainLabs/stylus-hello-world"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+description = "Stylus hello world example"
```
Comment (Jason-W123) [inline src/app/basic_examples/sending_ether/page.mdx:129]:
```suggestion
description = "Stylus sending ether example"
```

**Review by Jason-W123 — COMMENTED** [review]
Looks good to me and left some comments on tiny things, ty!

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 7]
```diff
@@ -0,0 +1,37 @@
+[package]
+name = "stylus-sending-ether"
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:6]: Thanks! One more thing here, should we change the repository url here?

File: example_code/basic_examples/sending_ether/Cargo.toml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+[dev-dependencies]
+tokio = { version = "1.12.0", features = ["full"] }
+ethers = "2.0"
+eyre = "0.6.8"
+
+[features]
+export-abi = ["stylus-sdk/export-abi"]
+
+[[bin]]
+name = "stylus-hello-world"
```
Comment (Jason-W123) [inline example_code/basic_examples/sending_ether/Cargo.toml:26]: Same here

File: src/app/basic_examples/sending_ether/page.mdx [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+### `Cargo.toml`
+
+```toml
+[package]
+name = "stylus-sending-ether"
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/OffchainLabs/stylus-hello-world"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
```
Comment (Jason-W123) [inline src/app/basic_examples/sending_ether/page.mdx:127]: Same here

**Review by Jason-W123 — APPROVED** [review]

File: example_code/basic_examples/sending_ether/Cargo.toml (hunk 7 above)
Comment (GreatSoshiant) [reply example_code/basic_examples/sending_ether/Cargo.toml]: Decided to remove the license from here, cause we already have it on another folder.

**Pull request merged**